scripts/ipfs/ipfs-swarm-cjdns.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Wait for ipfs to initalize
+attempts=15
+until [[ $( curl http://localhost:5001/api/v0/id -q 2>/dev/null) || ${attempts} -eq 0 ]]; do
+    sleep 1
+    attempts=$((attempts-1))
+done
+
+while read -r peer; do
+
```

Comment:
Can we please remove all these random newlines after a `while` and `if`? @darkdrgn2k you do this a lot, but also not all the time. It makes the code very inconsistent.